app/services/FileManager.scala

```diff
@@ -22,6 +22,7 @@ import scala.concurrent.Future
 
 trait FileManager {
   def copyToOutboundBucket(file: S3ObjectLocation): Future[Unit]
+  def copyToQuarantineBucket(file: S3ObjectLocation, details: String): Future[Unit]
```

Comment:
I would rename this method, to something more descriptive (create virus detection report?)